app/helpers/samples_helper.rb

```diff
@@ -109,6 +145,24 @@ def seek_cv_attribute_display(value, attribute)
     content
   end
 
+  def linked_custom_metadata_attribute_display(value)
+    html = ''
+     data = CustomMetadata.find(value.id).data
+    html += '<ul>'
+       CustomMetadata.find(value.id).custom_metadata_attributes.each do |attr|
```

Comment:
double calls to load `CustomMetadata.find(value.id)` - can just do once and store in a variable